src/map/battle.c

```diff
@@ -5308,7 +5308,7 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 #ifndef RENEWAL
 
 					if(sd->status.party_id && (temp=pc->checkskill(sd,TK_POWER)) > 0){
-						if( (i = party->foreachsamemap(party->sub_count, sd, 0)) > 1 ) // exclude the player himself [Inkfish]
+						if ((i = party->foreachsamemap(party->sub_count, sd, 0, sd->status.char_id)) > 0)
```

Comment:
I think if add new parameter, better add it in all calls to ``party->sub_count``

Reply:
Woops. Fixed & rebased.